src/parser.js

```diff
@@ -12,6 +12,7 @@ const parseCode = code => {
           allowSuperOutsideMethod: true,
           allowUndeclaredExports: true,
           tokens: true,
+          sourceType: "module",
```

Comment:
This fixed a random failure while running the transform that was caused by us using `import.meta` now.